src/api/bank-accounts/funds-transfers.md

```diff
@@ -270,4 +272,5 @@ A funds transfer represents either a top up to or a withdrawal from a Centrapay
 ```
 
 [direct debit endpoint]: {% link api/bank-accounts/bank-accounts.md %}#direct-debit-authority
+[settlement wallets]: {% link api/assets/wallets.md %}#settlement-wallets-experimental
```

Comment:
This link will break when we remove the experimental flag. Better to add a custom `<a name="settlement-wallets">` on the target page that won't break.

Reply:
Thanks, have updated that now.